### DISCUSSION TITLE
Invalid property path

### DISCUSSION THREAD

**Discussion by Pevalcar** [GodotNuts/GodotFirebase#157] [Q&A] [answered]
sorry my bad English
Hello,
I have a question, I want to create a database for an inventory, since when I remove or add a new Item the FireStore updates, my question is

1. when I use .update ("database", {Inventory}) it throws me {code: 400, message: Invalid property path "0"., Status: INVALID_ARGUMENT}. - when I use .remove () and then .add () they work.
2. I want to be able to delete Items without having to delete and re-upload the dataBase.

is it necessary to delete the Document, always? or this is not the best option for my project.

sorry my bad English

**Comment by BearDooks** [comment, marked as answer]
@Pevalcar forgive me for taking so long to get back to you. I just checked our most recent release and It seems to be working fine in my code. As an example I am doing this

_collection = Firebase.Firestore.collection('Firebasetester')
var add_task : FirestoreTask = _collection.add("Document2", {'name': 'Document2', 'active': 'true'})
var up_task : FirestoreTask = _collection.update("Document2", {'name': 'Document2', 'active': 'true', 'updated' : 'true'})

In doing this I can see the updates online for the document. Is this similar to what you are doing? Can you post your code so we ca take a look at it?

~ Chuck

**Reply by BearDooks** [reply]
@Pevalcar just checking in to see if you ever got this to work?

~ Chuck

**Reply by Pevalcar** [reply]
i can fix error.

creando un nuevo sistema {username:{inventario:{}...
banco:{}}}

Thank you for your attention
